py-polars/polars/io/excel/functions.py

```diff
@@ -251,6 +282,19 @@ def read_excel(
         )
 
 
+def _drop_null_columns(df: pl.DataFrame) -> pl.DataFrame:
+    # drop all-null cols with no name
```

Comment:
I'd say either make this a docstring, or even better: rename the function to `_drop_unnamed_null_columns` and you don't need a comment at all.

Reply:
Done 👍